frontend/src/api/auth/getAuth.tsx

```diff
@@ -0,0 +1,30 @@
+import { useMutation } from 'react-query';
+import Cookies from 'js-cookie';
+
+import { apiClientWrapper } from 'api/base';
+import { API } from 'constants/routes';
+import { COOKIE_VALUES } from 'constants/authConstants';
+
+const useGetAuth = () =>
+  useMutation(async (initialData: string | unknown) => {
+    let tokenResponse: {
```

Comment:
you can create tokenResopnse interface

Reply:
We have tokenResponse inside `useMutation`. Let's save this?